src/doc/rustdoc/src/SUMMARY.md

```diff
@@ -8,5 +8,5 @@
 - [Linking to items by name](linking-to-items-by-name.md)
 - [Lints](lints.md)
 - [Passes](passes.md)
-- [Advanced Features](advanced-features.md)
+- [Advanced features](advanced-features.md)
```

Comment:
Any reason you changed this?

Reply:
To be consistent with "Unstable features" and the capitalization in the other titles.